src/components/templates/ProductTemplate/ProductTemplateProgress/ProductTemplateProgress.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Progress, { ProgressProps } from '../../../molecules/Progress/Progress';
+import { minMedia } from '../../../../helpers/responsiveness';
+
+interface ProductTemplateProgressProps {
+  prevStepUrl?: string;
+  progress?: Pick<ProgressProps, 'currentStep' | 'totalSteps'>;
+  children: React.ReactNode;
+}
+
+const BackWrapper = styled.div`
+  ${minMedia.desktop`
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      left: 0;
+  `}
+`;
+
+const ProductTemplateProgressContainer = styled.div`
+  position: relative;
+  min-height: 30px;
+`;
+
+export function ProductTemplateProgress({ prevStepUrl, progress, children }: ProductTemplateProgressProps) {
+  if (!prevStepUrl || !progress) {
+    return null;
+  }
+
+  return (
+    <ProductTemplateProgressContainer data-automation="ZA.ProductTemplateProgress" className="l:mt-10 l:mb-4">
```

Comment:
The spacing for the progress container seems a bit large on desktop and small on mobile. I've attached references to some designs in the Jira ticket https://jira.zopa/browse/ZDS-41